### PR TITLE
Add brandmark logo (lightning app)

### PR DESCRIPTION
**Pull request by krshrimali**
## What does this PR do?

The logo wasn't visible before, has been added now. Preview: https://github.com/Lightning-AI/lightning/tree/lai/fix-brandmark/src/lightning_app

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the [CHANGELOG](https://github.com/Lightning-AI/lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [ ] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda

### CODE REVIEW

**Review by carmocca — APPROVED** [review]

**Review by Borda — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]
LGTM 

Moving forward, I suggest to use the s3 bucket for all images
https://pl-flash-data.s3.amazonaws.com/assets_lightning/docs/images/logos/lightning-ai.png

File: src/lightning_app/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div align="center">
 
-<img src="docs/source/_static/images/brandmark.png" width="400px">
+<img src="../../docs/source-app/_static/images/brandmark.png" width="400px">
```
Comment (krshrimali) [inline src/lightning_app/README.md:3]:
```suggestion
<img src="https://pl-flash-data.s3.amazonaws.com/assets_lightning/docs/images/logos/lightning-ai.png" width="400px">
```

**Review by krshrimali — COMMENTED** [review]
Thank you, @awaelchli - I agree! I've made the change in this PR only. Thanks for sharing the link.

**Comment by kaushikb11** [comment]
@Borda Let's have a new S3 bucket for storing assets? Rather than `pl-flash-data.s3`

**Comment by Borda** [comment]
In fact, I would include the logo in the codebase, this shall be rather small and close tight to the project itself... then we can move all larger files outside, but still moving outside make it challenging as we lose version support and so diffs across branches or release won't work...
cc: @Lightning-AI/core-lightning

**Comment by carmocca** [comment]
I believe the logo is showing up in master. Closing this!

**Pull request closed without merging**